testnets/galacticatestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "wardenprotocoltestnet",
+  "assets": [
+    {
+      "description": "The native token of Galactica Protocol Testnet",
+      "denom_units": [
+        {
+          "denom": "agnet",
+          "exponent": 0
+        },
+        {
+          "denom": "gnet",
+          "exponent": 18
+        }
+      ],
+      "base": "agnet",
+      "name": "Gnet",
+      "display": "gnet",
+      "symbol": "GNET",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/galactica/images/galactica.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/galactica/images/galactica.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/galactica/images/galactica.png",
```

Comment:
The svg and png have much different appearance. They may both be added as images, but belong in different image objects.